dashboard/app/views/home/index.html.haml

```diff
@@ -15,7 +15,7 @@
 - if current_user && current_user.teacher? && !current_user.accepted_latest_terms?
   - current_user.update_user_tos_version_accept()
   = render partial: 'layouts/implicit_new_user_terms_interstitial'
-- elsif SchoolInfoInterstitialHelper.show_confirmation_dialog?(current_user) || @force_school_info_confirmation_dialog
+- elsif (SchoolInfoInterstitialHelper.show_confirmation_dialog?(current_user) || @force_school_info_confirmation_dialog) && !params[:courseVersionId]
```

Comment:
Slightly hacky way of preventing both dialogs appearing at once. Let me know if you have a better idea on how to do this!